Extension/c_cpp_properties.schema.json

```diff
@@ -70,9 +70,22 @@
                         ]
                     },
                     "compileCommands": {
+                        "oneOf": [
+                            {
+                                "type": "string",
+                                "default": ""
+                            },
+                            {
+                                "type": "array",
+                                "items": {
+                                    "type": "string"
+                                },
+                                "uniqueItems": true,
+                                "default": []
+                            }
+                        ],
                         "markdownDescription": "Full path to `compile_commands.json` file for the workspace.",
```

Comment:
This description should be updated

Reply:
updated the description, feel free to suggest something different, English is not my native language.